1.9/usage/cli/command-reference/dcos-job/dcos-job-add.md

```diff
@@ -0,0 +1,35 @@
+---
+post_title: dcos job add
+menu_order: 0
+---
+    
+# Description
+Add a job.
+
+# Usage
+
+```bash
+dcos job add [OPTION]
+```
+
+# Options
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `--help, h`   |             |  Print usage. |
+| `--info`   |             |  Print a short description of this subcommand. |
+| `--version, v`   |             | Print version information. |
+
+# Positional arguments
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `<job-file>`   |             |  A JSON-formatted job definition. |
+
+# Parent command
+
+| Command | Description |
+|---------|-------------|
+| [dcos job](/docs/1.9/usage/cli/command-reference/dcos-job/) |  Deploy and manage jobs in DC/OS. |
+
+# Examples
```

Comment:
I think I will point to the more extensive examples here: https://docs.mesosphere.com/1.9/usage/jobs/examples/#create-job. What do you think?